packages/fxa-content-server/app/scripts/views/index.js

```diff
@@ -203,6 +212,14 @@ class IndexView extends FormView {
     return firefoxMail.test(email);
   }
 
+  _isEmailRelayDomain(email) {
+    // "@mozmail.com" email addresses are not valid
+    // at this time, therefore block the attempt.
+    // the added 'i' disallows uppercase letters
+    const relayMail = new RegExp('@mozmail.com$', 'i');
```

Comment:
The `.` needs to be escaped in this case.  (Personally I prefer `string.endswith` for a simple match like this.)